src/components/atoms/Dropdown/index.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+interface Props {
+  options: { name: string; id: number }[];
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  placeholder: string;
+  styleType: "big" | "small";
+  selectedOptionId?: number;
+}
+
+function Dropdown({ options, onChange, placeholder, styleType, selectedOptionId }: Props) {
+  const styleObj = {
+    small: "rounded-full text-center p-1 w-[150px]",
+    big: "rounded-3xl p-4 w-[180px] text-xl text-center shadow-lg",
+  };
+  return (
+    <select
+      onChange={onChange}
+      className={`border text-neutral-500 appearance-none ${styleObj[styleType]}`}
+      value={typeof selectedOptionId !== undefined ? selectedOptionId : -1}
```

Comment:
selectedOptionId가 optional props인데 이 값을 주지 않은 경우에 -1로 주고 있어요
아래와 같이 default value를 정해주면 좀 더 읽기 쉬워질 것 같습니다 :)

```suggestion
function Dropdown({ options, onChange, placeholder, styleType, selectedOptionId = -1 }: Props) {
  const styleObj = {
    small: "rounded-full text-center p-1 w-[150px]",
    big: "rounded-3xl p-4 w-[180px] text-xl text-center shadow-lg",
  };
  return (
    <select
      onChange={onChange}
      className={`border text-neutral-500 appearance-none ${styleObj[styleType]}`}
      value={selectedOptionId}
```

Reply:
리팩토링 완료했습니다.